### PR TITLE
Update DEVCATALOG_toolkit.md

### PR DESCRIPTION
**Pull request by hamham223**
update readme

### CODE REVIEW

**Comment by hamham223** [comment]
use `intel-tensorflow==2.9.1` or `tensorflow==2.9.0`? it's not consistent

**Comment by hamham223** [comment]
@hkvision  take a 👀 and 🙄 ?

File: python/friesian/DEVCATALOG_toolkit.md [hunk 1]
```diff
@@ -196,15 +196,22 @@ python train_2tower.py \
     --data_dir ../../../../apps/wide-deep-recommendation/recsys_data/preprocessed \
     --model_dir recsys_2tower/ \
     --batch_size 8000
+
+python predict_2tower.py \
+    --backend spark \
+    --executor_cores 8 \
+    --executor_memory 6g \
+    --model_dir recsys_2tower/
```
Comment (hamham223) [inline python/friesian/DEVCATALOG_toolkit.md:207]: WIP: need change the args

File: python/friesian/DEVCATALOG_toolkit.md [hunk 2]
```diff
@@ -196,15 +198,23 @@ python train_2tower.py \
...
 ```
 
 **Expected Training Workflow Output**
 
 Check out the processed data and saved models after the training:
 ```
+cd /path/to/BigDL
 ll apps/wide-deep-recommendation/recsys_data/preprocessed
 ll recsys_wnd/
```
Comment (hkvision) [inline python/friesian/DEVCATALOG_toolkit.md:216]: ??

Comment (hamham223) [reply python/friesian/DEVCATALOG_toolkit.md:216]: user might in another directory when installing redis

**Pull request merged**